app/models/forem_instance.rb

```diff
@@ -0,0 +1,9 @@
+class ForemInstance
+  def self.deployed_at
+    ApplicationConfig["RELEASE_FOOTPRINT"].presence || ENV["HEROKU_RELEASE_CREATED_AT"].presence
+  end
+
+  def self.latest_commit_id
+    ApplicationConfig["FOREM_BUILD_SHA"].presence || ENV["HEROKU_SLUG_COMMIT"].presence
```

Comment:
For the Forem.cloud environment FOREM_BUILD_SHA will be set whereas for a Heroku environment HEROKU_SLUG_COMMIT will be set

Reply:
same thing about moving this to a frozen constant

Reply:
✅